src/platform/user/profile/vet360/containers/Vet360ProfileField.jsx

```diff
@@ -106,7 +106,13 @@ class Vet360ProfileField extends React.Component {
   };
 
   onSubmit = () => {
-    this.captureEvent('update-button');
+    // The validateAddress thunk will handle its own dataLayer additions
+    if (this.props.useAddressValidation) {
+      if (!this.props.fieldName.toLowerCase().includes('address')) {
+        this.captureEvent('update-button');
```

Comment:
Do we want to escape early here? Or maybe use an `else`? Otherwise `captureEvent('update-button')` could hypothetically be called twice I believe! 🙂